monoio-http-client/src/client/connector.rs

```diff
@@ -180,7 +181,7 @@ impl HttpConnector {
         };
 
         match proto {
-            Version::HTTP_11 => Ok(HttpConnection::H1(ClientCodec::new(io))),
+            Version::HTTP_11 => Ok(HttpConnection::H1(ClientCodec::new(io, Duration::from_secs(60)))),
```

Comment:
Don't hard code 60s. And maybe add another function like `connect_with_timeout`?